.github/workflows/pr-build.yml

```
@@ -0,0 +1,40 @@
+
+jobs:
+  test-build:
+    runs-on: ubuntu-latest
+
+    environment: SPRING_BOOT
+
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Set up JDK 17
+        uses: actions/setup-java@v3
```

actionlint (1.7.4)

the runner of "actions/setup-java@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)